src/Observable.ts

```diff
@@ -258,3 +258,9 @@ export class Observable<T> implements CoreOperators<T>  {
   zip: <R>(...observables: Array<Observable<any> | ((...values: Array<any>) => R)>) => Observable<R>;
   zipAll: <R>(project?: (...values: Array<any>) => R) => Observable<R>;
 }
+
+//import after Observable<T> is declared and exported
+import {ArrayObservable} from './observable/fromArray';
+
+Observable.fromArray = ArrayObservable.create;
```

Comment:
You're correct, why I did import `fromArray` only except `from`? :( Since it's same module I also think leaving `fromArray` would be fine, updating to include `from` actually..